common/subdevice-model.cpp

```diff
@@ -1419,6 +1419,45 @@ namespace rs2
         return is_cal_format;
     }
 
+    bool subdevice_model::is_depth_calibration_profile() const
```

Comment:
Since on a device that is not D555 this will return false even if the res is calibration profile, maybe you should consider renaming it?
 `block_depth_calibration_profile_if_needed` or something like that?

Reply:
On other (current) devices this kind of profile is not exposed and cannot be selected.
I have tried to match the name of the existing `is_ir_calibration_profile` function name (that, BTW, will also return false if not a specific device D405) .
If you still think we should rename then I will.